app/src/main/java/com/github/featuredetectandroid/ui/Menu.kt

```diff
@@ -0,0 +1,77 @@
+package com.github.featuredetectandroid.ui
+
+import androidx.compose.foundation.layout.Arrangement
+import androidx.compose.foundation.layout.Row
+import androidx.compose.foundation.layout.fillMaxWidth
+import androidx.compose.foundation.layout.padding
+import androidx.compose.foundation.selection.selectable
+import androidx.compose.material.RadioButton
+import androidx.compose.material.Text
+import androidx.compose.runtime.Composable
+import androidx.compose.runtime.mutableStateOf
+import androidx.compose.runtime.remember
+import androidx.compose.ui.Alignment
+import androidx.compose.ui.Alignment.Companion.CenterVertically
+import androidx.compose.ui.Modifier
+import androidx.compose.ui.text.style.TextAlign
+import androidx.compose.ui.unit.dp
+import com.github.featuredetectandroid.ui.theme.Typography
+import com.github.featuredetectandroid.utils.algorithmsNamesAsList
+
+@Composable
+fun Menu(currentAlgorithm: String, changeSelectedAlgorithmIntPreferences: (String) -> Unit) {
+    val radioOptions = algorithmsNamesAsList()
+    val (selectedAlgorithm, onAlgorithmSelection) = remember { mutableStateOf(currentAlgorithm) }
+
+    Row(horizontalArrangement = Arrangement.Center, verticalAlignment = Alignment.Bottom) {
+        Text(
+            text = "Keypoint detection algorithm:",
+            modifier = Modifier.padding(all = 20.dp),
+            textAlign = TextAlign.Start,
+            style = Typography.body1
```

Comment:
It should probably named `h6` (in the theme, I mean) as this is a relatively small header, not a body text. You can find examples [here](https://material.io/design/typography/the-type-system.html#type-scale).

Reply:
Also, don't use the elements from your theme like this -- it should be `MaterialTheme.typography.body1`. The whole point of the theme is that the UI doesn't depend on concrete theme classes, it just uses the values provided in the theme it is launched in.

Reply:
fixed